cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsStackRequestHelper.java

```diff
@@ -144,6 +145,21 @@ private String getRootDeviceName(AuthenticatedContext ac, CloudStack cloudStack)
             parameters.add(new Parameter().withParameterKey("PortParameter").withParameterValue(Objects.toString(port)));
         }
 
+        if (awsRdsInstanceView.getVPCSecurityGroups().isEmpty()) {
+            parameters.addAll(
+                    asList(
+                            new Parameter().withParameterKey("VPCIdParameter").withParameterValue(String.valueOf(awsNetworkView.getExistingVPC())),
```

Comment:
I recommend adding a comment here that the VPC is guaranteed to be present in the stack because of NetworkParameterFactoryService. That isn't obvious.

Reply:
Ok, the code was not quite ok, VpcId and cidr could be missing if network is provided but sec group not in the request => I now always add the vpc parameters.